src/helpers/data/plugins/i18n/locales/it.json

```diff
@@ -139,326 +81,377 @@
+        "favorite": "Stai per cancellare {modelName} dai preferiti.",
+        "bookmark": "Stai per cancellare {modelName} dai salvati.",
+        "post": "Stai per cancellare questo post."
       },
-      "streamable": "In streaming",
-      "other": "Altro",
-      "albumTypes": {
-        "album": "Album",
-        "albumVarious": "Album A/V",
-        "group": "Gruppo di album",
-        "track": "Brano"
+      "also": {
+        "library": {
+          "artist": "Verranno inoltre cancellati tutti gli album e i brani di questo artista.",
+          "album": "Verranno inoltre cancellati tutti i brani di questo album."
+        },
+        "playlist": "Verranno inoltre cancellati tutti i brani nella playlist."
       },
-      "albumsTypes": {
-        "album": "Album",
-        "single": "Singoli",
-        "single_ep": "Singoli / EP",
-        "compilation": "Compilation",
-        "video": "Video",
-        "misc": "Varie",
-        "appearance": "Apparizioni"
+      "undo": "Questa azione non potrà essere annullata.",
+      "sure": "Continuare?"
+    }
+  },
+  "forms": {
+    "fields": {
+      "email": "Email",
+      "password": "Password",
+      "passwordConfirmation": "Conferma password",
+      "nickname": "Nome utente",
+      "title": "Titolo",
+      "image": "Carica immagine",
+      "gender": "Genere",
+      "birthdate": "Compleanno",
+      "country": "Nazione",
+      "city": "Città",
+      "genders": {
+        "male": "Maschio",
+        "female": "Femmina",
+        "other": "Altro"
       }
     },
-    "add": {
-      "account": {
-        "import": {
-          "lastfm": "Importa dati da Last.FM"
-        }
+    "remember": "Ricorda dati",
+    "extra": "Altre informazioni",
+    "errors": {
+      "notFound": "Email o password errati",
+      "empty": {
+        "email": "L'email non può essere vuota",
+        "password": "La password non può essere vuota",
+        "passwordConfirmation": "Conferma password non può essere vuota",
+        "nickname": "Il nome utente non può essere vuoto",
+        "title": "Il titolo è vuoto"
       },
-      "folder": {
-        "select": "Seleziona cartella"
+      "invalid": {
+        "email": "Email non valida"
       },
-      "import": {
-        "active": {
-          "tracks": "Sono stati importati {value} su {total} brani...",
-          "plays": "Sono state importate {value} di {total} riproduzioni..."
-        },
-        "success": {
-          "tracks": {
-            "header": "Tutti i {count} brani sono stati importati con successo",
-            "content": "È possibile controllarli prima di salvarli nella libreria"
-          }
-        },
-        "error": {
-          "files": "Impossibile importare {count} file. Correggere i dati ID3 e riprovare:"
-        }
+      "taken": {
+        "email": "L'email è già utilizzata",
+        "nickname": "Il nome utente è già utilizzato"
```

Comment:
@starise I've noticed that 'title' key is missing here.
Add it please in the next PR.